src/node/context.h

```diff
@@ -40,7 +41,11 @@ struct NodeContext {
     std::unique_ptr<BanMan> banman;
     ArgsManager* args{nullptr}; // Currently a raw pointer because the memory is not managed by this struct
     std::unique_ptr<interfaces::Chain> chain;
+    //! List of all chain clients (wallet processes or other client) connected to node.
```

Comment:
```suggestion
    //! List of all chain clients (wallet client or other client) connected to node.
```

nit in e4f4350:

could simply say that there is at most a single wallet client in this list, not one for each wallet.